ftdetect/mako.vim

```diff
@@ -1 +1,11 @@
+if !exists("g:mako_detect_lang_from_ext")
+  let g:mako_detect_lang_from_ext = 1
+endif
+if g:mako_detect_lang_from_ext
+  au BufNewFile *.*.mako   execute "do BufNewFile " . expand("<afile>:r") | let b:mako_outer_lang = &filetype
```

Comment:
Good point. It still works when restricted to filetypedetect, so I've made that change.